packaging/msi/IceServices.wixproj

```diff
@@ -12,8 +12,6 @@
     <VCInstallDir Condition="'$(VCInstallDir)' != ''">$([MSBuild]::EnsureTrailingSlash('$(VCInstallDir)'))</VCInstallDir>
     <!-- Use DefineConstants to make MSBuild properties available in wxs Wix Files-->
     <DefineConstants>Version=3.8.0</DefineConstants>
-    <DefineConstants>$(DefineConstants);IceManualUrl=https://docs.zeroc.com/ice/latest/cpp/</DefineConstants>
-    <DefineConstants>$(DefineConstants);IceSourceUrl=https://github.com/zeroc-ice/ice</DefineConstants>
```

Comment:
These links aren't used anymore, they were part of the start menu entries in previous versions.